pkgs/tools/security/melt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "melt";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "charmbracelet";
+    repo = "melt";
+    rev = "v${version}";
+    sha256 = "sha256-HambmUodAwgi1/r/Nj7P7gXNK5pyBO3omMZ9ZvSK7cc=4";
+  };
+
+  vendorSha256 = "sha256-1f3QBbWaTiplEp/4ihds1PwrEnoq/81TzlT46TufGFs=";
+
+  doCheck = false;
```

Comment:
Could you please add a comment why they tests are not executed?

Reply:
Good catch 👍! Took this over from a template, there's no reason not to run the tests here

Reply:
`doCheck = true;` is the default. Thus, could be removed.